assets/app/scripts/controllers/create/nextSteps.js

```diff
@@ -16,21 +16,53 @@ angular.module("openshiftConsole")
     $scope.emptyMessage = "Loading...";
     $scope.alerts = [];
     $scope.loginBaseUrl = DataService.openshiftAPIBaseUrl();
-    $scope.projectName = $routeParams.project;
     $scope.buildConfigs = {};
 
+    $scope.projectName = $routeParams.project;
+    imageName = $routeParams.imageName;
+    imageTag = $routeParams.imageTag;
+    namespace = $routeParams.namespace;
+
+    var imageName = $routeParams.imageName;
+    var imageTag = $routeParams.imageTag;
+    var namespace = $routeParams.namespace;
+
+    var name = $routeParams.name;
+    var nameLink = "";
+    if (creatingFromImage()) {
+      nameLink = "project/" + $scope.projectName + "/create/fromimage?imageName=" + imageName + "&imageTag=" + imageTag + "&namespace=" + namespace + "&name=" + name;
+    } else {
+      nameLink = "project/" + $scope.projectName + "/create/fromtemplate?name=" + name + "&namespace=" + namespace;
+    }
+
+    $scope.breadcrumbs = [
+      {
+        title: $scope.projectName,
+        link: "project/" + $scope.projectName
+      },
+      {
+        title: "Add to Project",
+        link: "project/" + $scope.projectName + "/create"
+      },
+      {
+        title: name,
+        link: nameLink
+      },
+      {
+        title: "Next Steps"
```

Comment:
This is a little different than the previous behavior that said "Creating" if all tasks weren't complete. I think I'm OK with that, though.